### ISSUE TITLE
Hold down times/out of order packets

### ISSUE THREAD

**Issue by MaxF12** [GrumpyOldTroll/ietf-dorms-cluster#4]
Assume that a manifest gets significantly delayed  so the data packets sit and wait for their 2 second hold out. In the mean time, a manifest arrives that verifies newer packets. Shortly before the hold down time of the old packets runs out, the manifest arrives, verifying all packets that were waiting at once. This would lead to a burst of out-of order packets arriving at the application. To me that sounds like it could potentially cause trouble, as applications would need to take special precautions to deal with such bursts. 

Considering there is focus on making sure manifests arrive before data packets, is there really any point in keeping data packets without a corresponding manifest around at all? Won't such packets be irrelevant for most multicast applications anyway if they only get verified 2 seconds after they arrive?

**Comment by GrumpyOldTroll** [comment]
Yes, good point.  I think this is not a sustainable requirement and we should remove it, as with the closely related #6.
